_posts/2020-02-10-hhvm-4.44.markdown

```diff
@@ -5,7 +5,7 @@ author: fred
 category: blog
 ---
 
-HHVM 4.44 is released! This release marks the end of support for 4.37;
```

Comment:
4.37 doesn't exist :)

This should probably have been 4.38 but I didn't want to retroactively announce the end of support for a different version so instead I announced the end of support for two versions this week (I'm planning to support both until the end of today though).